### PR TITLE
Feature/artwork information page

### PR DESCRIPTION
**Pull request by MonaHer**
I created the `ArtworkInfo` component and it is rendered on `pages/artwork-info/[id].js`. 
The `ArtworksList` component is rendered on the `pages/index.js`. 

After second code review: Added pagination and cleaned up the code a bit

### CODE REVIEW

File: components/ArtworkInfo/index.js [hunk 1]
```diff
@@ -0,0 +1,85 @@
+import Image from "next/image";
+
+export default function ArtworkInfo({ artworks }) {
+  return (
+    console.log(artworks),
```
Comment (vaupunkt) [inline components/ArtworkInfo/index.js]: you don't need the console.log here :)

File: pages/artwork-info/[id].js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+  if (!currentArtworkInfo) {
+    return null;
+  }
+
+  return (
+    <>
+      <Link href={`/`}>Go back to Artworkslist</Link>
+      <h1>ArtworkInfoPage</h1>
+      <ArtworkInfo artworks={{ items: [currentArtworkInfo] }} />
```
Comment (vaupunkt) [inline pages/artwork-info/[id].js]: i think you are making it a bit more complicated than you have to :) 

this should work here and you can simplify the ArtworkInfo Component
```suggestion
      <ArtworkInfo artworks={currentArtworkInfo} />
```

File: components/ArtworkInfo/index.js (hunk 1 above)
Comment (vaupunkt) [reply components/ArtworkInfo/index.js]: also you would put the console.log above the return

File: components/ArtworkInfo/index.js [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+                  <p>{labelText}</p>
+                </li>
+              );
+            }
+          )}
+        </ul>
+      </>
+    )
+  );
+}
```
Comment (vaupunkt) [inline components/ArtworkInfo/index.js]: This should also work and is a bit better readable :)

```suggestion
export default function ArtworkInfo({ artworks }) {
  const {
    id,
    image_thumbnail,
    production,
    dimensions,
    titles,
    current_location_name,
    labels,
    production_dates_notes,
  } = artwork;
  const {
    creator_forename,
    creator_surname,
    creator_date_of_birth,
    creator_date_of_death,
  } = production;
  const creatorLifeDates = `${
    creator_date_of_birth ? creator_date_of_birth.slice(0, 4) : ""
  }-${creator_date_of_death ? creator_date_of_death.slice(0, 4) : ""}`;

  const dimension1 =
    dimensions && dimensions[0] && dimensions[0].value
      ? dimensions[0].value
      : "";
  const dimension2 =
    dimensions && dimensions[1] && dimensions[1].value
      ? dimensions[1].value
      : "";

  const titleText =
    titles && titles[0] && titles[0].title ? titles[0].title : "";
  const labelText = labels && labels[0] && labels[0].text ? labels[0].text : "";

  const productionDatesNotes =
    production_dates_notes && production_dates_notes[0];
  const prefix = "Værkdatering: ";
  const formattedProductionDatesNotes =
    productionDatesNotes && productionDatesNotes.startsWith(prefix)
      ? productionDatesNotes.slice(prefix.length)
      : productionDatesNotes;
  return (
    <>
             <Image src={image_thumbnail} alt={titleText} height={300} width={300} />
        <p>
          Title: {titleText},{formattedProductionDatesNotes}
        </p>
        <p>
          Creator: {creator_forename} {creator_surname},{creatorLifeDates}
        </p>
        <p>
          📐
          {dimension1}x{dimension2}
          cm
        </p>
        <p>📍 {current_location_name}</p>
        <p>{labelText}</p>
    </>
  );
}

```

File: pages/index.js [hunk 4]
```diff
@@ -1,10 +1,10 @@
-import Artworks from "@/components/Artworks";
+import ArtworksList from "@/components/ArtworksList/index.js";
 
-export default function HomePage() {
+export default function HomePage({ artworks }) {
   return (
     <>
       <h1>SKM Notes</h1>
```
Comment (vaupunkt) [inline pages/index.js]: the museum is SMK but your app is called SKM - is this a typo? 😅

**Review by vaupunkt — CHANGES_REQUESTED** [review]
Hey :) 
I think you made it a bit harder for yourself than you needed to. I tried to make it a bit more simple - i hope it helps 🙏🏻

File: pages/_app.js [hunk 5]
```diff
@@ -1,10 +1,40 @@
...
+  return res.json();
+};
 
 export default function App({ Component, pageProps }) {
+  const {
+    data: artworks,
+    error,
+    isLoading,
+  } = useSWR(
+    "https://api.smk.dk/api/v1/art/search/?keys=*&fields=image_thumbnail&fields=titles&fields=id&fields=production&fields=dimensions&fields=current_location_name&fields=production_dates_notes&fields=labels&filters=[image_hq:true],[object_names:painting],[public_domain:true]&offset=0&rows=2000&lang=en",
```
Comment (actyvystom) [inline pages/_app.js]: Is there a way to limit the number of returned objects from the API? The list is endlessly long, you should think of a way to implement some kind of pagination (probably controlled via offset and rows parameter in the request).

**Review by actyvystom — CHANGES_REQUESTED** [review]
Nice work! 👍🏻 
See some comment on reducing the amount of results / implement some pagination.
Cheers,
Thomas

File: pages/_app.js (hunk 5 above)
Comment (MonaHer) [reply pages/_app.js]: Good idea! 😊 Thank you!

Comment (VolkerJacobsen) [reply pages/_app.js]: Wow! I am impressed. Well done, Mona!

**Pull request merged**